apps/web/app/WithEmbedSSR.tsx

```diff
@@ -14,10 +14,6 @@ export default function withEmbedSsrAppDir<T extends Record<string, any>>(
 ) {
   return async (context: GetServerSidePropsContext): Promise<T> => {
     const { embed, layout } = context.query;
-    const isCOEPEnabled = context.query["flag.coep"] === "true";
-    if (isCOEPEnabled) {
-      context.res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
-    }
```

Comment:
@Udit-takkar @sean-brydon Removed this header from here. With App Router, I think middleware is the only place it can go. So, I have added the logic there.

It loads the first page correctly in embed but navigation is still blocked.

Reply:
Also see this https://www.loom.com/share/2b6dcdcfc9fb43e99808ac5ca420e3cb